Capri.Services/Promoters/PromoterDeleter.cs

```diff
@@ -25,10 +25,17 @@ await _context
             if(promoter == null)
             {
                 return ServiceResult<Promoter>.Error(
-                    "Promoter with the given id does not exist");
+                    $"Promoter with id {id} does not exist");
             }
 
+            var applicationUser = 
+                await _context
+                .Users
+                .FirstOrDefaultAsync(u => u.Id == promoter.UserId);
+
             _context.Promoters.Remove(promoter);
+            _context.Users.Remove(applicationUser);
```

Comment:
Shouldn't you use userManager for removing ApplicationUser?

Reply:
I'm not sure. In UserCreator we don't use UserManager.